### ISSUE TITLE
[tooltip] NVDA + FF doesn't announce the `aria-describedby` attribute if it's dynamically inserted

### ISSUE THREAD

**Issue by chaance** [reach/reach-ui#658]
Details explained here by @jeryj: https://twitter.com/juryjowns/status/1296947347499487233?s=20

- NVDA + Firefox doesn't announce the `aria-describedby` attribute if it's dynamically inserted
